templates/ai/index.html

```diff
@@ -0,0 +1,688 @@
+      <ul class="p-inline-images--small u-no-margin--bottom u-align-text--left">
+        <li class="p-inline-images__item u-no-margin--left u-no-margin--bottom">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/38697bf3-server_brand-dark-grey.svg",
+            alt="Server brand",
+            width="46",
+            height="50",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin--left u-no-margin--bottom">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/7ecbaf72-OpenStack_openstack-logo.svg",
+            alt="Openstack",
+            width="51",
+            height="50",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin--left u-no-margin--bottom">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg",
+            alt="AWS",
+            width="84",
+            height="50",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+        </li>
+      </ul>
+      <ul class="p-inline-images--small u-no-margin--bottom u-align-text--left">
+        <li class="p-inline-images__item u-no-margin--left u-no-margin--bottom">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/e5f6bd10-Google_Cloud_Platform-Logo.png",
+            alt="Google Cloud",
+            width="223",
+            height="35",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item u-no-margin--left u-no-margin--bottom">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/da9a1344-Microsoft-Azure-logo_stacked_transparent.png",
+            alt="Microsoft Azure",
+            width="122",
+            height="35",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Every Kubeflow operator you need</h1>
+      <h4>Lifecycle management of your data stack on Kubernetes</h4>
+      <ul>
+        <li>Make use of open source lifecycle management code and compound the benefit of
+          <a href="http://charmhub.io">200+ charm operators</a>.</li>
+        <li>Composable operators for data scientists to stand up and integrate the applications they need, on a laptop, workstation or cluster.</li>
+        <li>Install, configure, upgrade, remove.</li>
+      </ul>
+      <p>
+        <a href="http://charmed-kubeflow.io" class="p-button--positive p-link--external">Kubeflow operators</a>
+        <a href="/ai/contact-us?product=ai-index">Contact us&nbsp;&rsaquo;</a>
```

Comment:
Check this comment https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit?disco=AAAALCK26VU, it's a different contact us